### ISSUE TITLE
Small Steam Icons are incorrect aspect ratio

### ISSUE THREAD

**Issue by marmite22** [PhilipK/BoilR#66]
Steamdeck, Heroic/Flatpack/Epic, BoilR v.0.9.22 using SteamGrid api key

The images downloaded for the shortcuts in the small library on the left are the incorrect aspect ratio.

Also some things like Emulators have automatically picked up artwork for the banner/large icons but have no icon for the small library (e.g. Dolphin in my screenshot)

![image](https://user-images.githubusercontent.com/2477900/161242593-b7b37904-ae69-42a3-978e-0ad04e1a46bc.png)

**Comment by PhilipK** [comment]
This is a "feature" for big picture mode. The big picture mode uses the icon images as its "cover images".
I am thinking that this has to be implemented as a "enable big picture support" mode, a mode that is off by default.
Then big picture mode will look bad, but it will look better with the icons.

**Comment by PhilipK** [comment]
@marmite22 you know the drill by now :-) There is a new release:
https://github.com/PhilipK/BoilR/releases/tag/v.0.9.23

Can you test if it works?

**Comment by marmite22** [comment]
In the settings, optimize_for_big_picture = false 

and now I don't get icons in the small library OR in big picture mode (but they still work in game mode though).

**Comment by marmite22** [comment]
with the settings set to optimize_for_big_picture = true
 
 I have got correct icons in the small library but then it also displays those same small icons (but stretched/blurry) in big picture.

**Comment by PhilipK** [comment]
ah found the problem, i check if the icon exists before i download it... so if you just run it twice it will work.... I will get it fixed in the next release.

**Comment by PhilipK** [comment]
https://github.com/PhilipK/BoilR/releases/tag/v.0.9.24 should fix the icons

**Comment by marmite22** [comment]
I'm not sure exactly what my set of steps were but I was testing both `optimize_for_big_picture = false` and `optimize_for_big_picture = true` and then testing both big picture and the library view and I've managed to get it to a point where BOTH look perfect. The big picture view has the nice wide aspect ratio images and the little library has got nice sharp little icons - and I've just checked and they look great in game mode too.

The last time I ran it was with `optimize_for_big_picture = false` but I don't know if the fact that I ran it the other way first is why it's working or if it would be working if I'd just run it with false the first time.

Anyway, it seems there is some way to have it both ways rather than either/or.

I'm afraid I'm not around much over the weekend for more testing - but for me at least this seems to be working now so thanks again.
